### PR TITLE
Minor code cleanup in error handling.

### PR DESCRIPTION
**Pull request by tariq1890**

### CODE REVIEW

File: handlers/check.go [hunk 1]
```diff
@@ -32,8 +32,7 @@ func CheckHandler(w http.ResponseWriter, r *http.Request) {
 	repo, err := download.Clean(r.FormValue("repo"))
 	if err != nil {
 		log.Println("ERROR: from download.Clean:", err)
-		w.WriteHeader(http.StatusBadRequest)
-		w.Write([]byte(`Could not download the repository: ` + err.Error()))
+		http.Error(w, `Could not download the repository: `+err.Error(), http.StatusBadRequest)
```
Comment (shawnps) [inline handlers/check.go]: @tariq1890 can we make these "  rather than ` too? I don't think there was any reason to use ` here before

**Comment by tariq1890** [comment]
@shawnps I have updated the PR with your review comment.

**Comment by shawnps** [comment]
Thanks @tariq1890

**Comment by tariq1890** [comment]
Hi @shawnps . Has this code change been deployed in the production?

**Comment by shawnps** [comment]
@tariq1890 hmm, I thought so...let me restart the service just in case

**Pull request merged**